source/preface_third.Rmd

```diff
@@ -25,111 +25,87 @@ The book in your hands, or on your screen, is the third edition of a book
-
-But we needed both of these insights to get anywhere.   We cannot hope to teach
-two hard subjects at the same time; mathematics, and statistical reasoning.
-That is what Simon has done: he took away the mathematics, and replaced it with
-something that is much easier to reason about; then he can concentrate on the
+Simon gives the early history of this book in original preface
+(\@ref(brief-history)).  He starts with the following observation:
+
+> In the mid-1960's, I noticed that most graduate students — among them many
+> who had had several advanced courses in statistics — were unable to apply
+> statistical methods correctly...
+
+Simon then applied his striking capacity for independent thought to the
+problem — and came to two essential conclusions.
+
+His first conclusion was that introductory courses in statistics use far too
+much mathematics. Most students cannot follow that
+mathematics, and they quickly get lost, reducing the subject to  "mumbo-jumbo"
+(\@ref(brief-history)).
+
+On its own, this was not a new conclusion.  Simon quotes a classic textbook by
+Wallis and Roberts [-@wallis1957statistics], in which they compare teaching
+statistics through mathematics to teaching in a foreign language.  More
+recently, other teachers of statistics have come to the same conclusion. Cobb
+[-@cobb2007introductory] argues that it is practically impossible to teach
+students the level of mathematics they would need to understand standard
+introductory courses.  It turns out (\@ref(resampling-data-science)) that he
+also agrees with Simon about the solution.
+
+Simon's great contribution was to see *how* we can replace the mathematics,
+to better reveal the true heart of statistical thinking.  His starting point
+appears in the original preface: "Beneath the logic of a statistical inference
+there necessarily lies a physical process". Drawing conclusions from noisy
+data means building a *model* of the noisy world, and seeing how that model
+behaves. That model can be physical, where we generate the noisiness of the
+world using physical devices like dice and spinners and coin-tosses. In fact,
+Simon used exactly these kinds of devices in his first experiments
+in teaching [@simon1969basic]. He then saw that it was much more efficient to
+build these models with simple computer code, and the result was the first and
+second editions of this book, with their associated software, the *Resampling
+Stats* language — of which more later.
+
+Simon's second conclusion follows from the first. Now that Simon had stripped
+away the unnecessary barrier of mathematics, he had got to the heart of what
+is interesting and difficult in statistics.  Drawing conclusions from noisy
+data involves a lot of hard, clear thinking.  We need to be honest with our
+students about that; statistics is hard, not because it is obscure (it need
+not be), but because it deals with difficult problems. It is exactly that hard logical thinking that can make statistics so
```

Comment:
"exactly that hard" is an awkward phrase.  Can we replace it with something else?